src/Microsoft.AspNetCore.Server.Kestrel/Filter/LoggingFilterKestrelOptionsExtensions.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNetCore.Server.Kestrel.Filter
```

Comment:
We can bring this into the Microsoft.AspNetCore.Server.Kestrel namespace with the options type. We just didn't want it in the Microsoft.AspNetCore.Builder namespace.